src/explore-education-statistics-frontend/src/components/CookieBanner.tsx

```diff
@@ -26,24 +26,55 @@ function CookieBanner({ wide }: Props) {
           'dfe-width-container--wide': wide,
         })}
       >
-        <p>
-          <span>GOV.UK uses cookies to make the site simpler.</span>{' '}
-          <ButtonText
-            onClick={() => {
-              setBannerSeenCookie(true);
-              if (typeof getCookie('disableGA') === 'undefined') {
-                setGADisabledCookie(false);
-              }
-            }}
-          >
-            Accept Cookies
-          </ButtonText>{' '}
-          or{' '}
-          <Link to="/cookies">
-            find out more about cookies and cookie settings
-          </Link>
-          .
-        </p>
+        <div
+          className="govuk-cookie-banner "
+          role="region"
+          aria-label="Cookies on Explore Education Statistics"
+        >
+          <div className="govuk-cookie-banner__message govuk-width-container">
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-two-thirds">
+                <h2 className="govuk-cookie-banner__heading govuk-heading-m">
+                  Cookies on Explore Education Statistics
+                </h2>
+
+                <div className="govuk-cookie-banner__content">
+                  <p>
+                    We use some essential cookies to make this service work.
+                  </p>
+                  <p>
+                    We’d also like to use analytics cookies so we can understand
+                    how you use the service and make improvements.
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <div className="govuk-button-group">
+              <Button
+                onClick={() => {
+                  setBannerSeenCookie(true);
+                  if (typeof getCookie('disableGA') === 'undefined') {
+                    setGADisabledCookie(false);
+                  }
+                }}
+              >
+                Accept analytics cookies
```

Comment:
We need to respond to the user accepting/rejecting the cookies as per this section in the design system:

https://design-system.service.gov.uk/components/cookie-banner#when-the-user-accepts-or-rejects-cookies-using-the-cookie-banner